apps/passport-server/test/generic-issuance/pipelines/auto-issuance/autoIssuanceSpec.ts

```diff
@@ -0,0 +1,1089 @@
+import { getEdDSAPublicKey, newEdDSAPrivateKey } from "@pcd/eddsa-pcd";
+import { expectIsEdDSATicketPCD } from "@pcd/eddsa-ticket-pcd";
+import { EmailPCDPackage } from "@pcd/email-pcd";
+import {
+  PipelineLogLevel,
+  PodboxTicketActionResponseValue,
+  PretixPipelineDefinition,
+  createCredentialPayload,
+  requestGenericIssuanceSemaphoreGroup,
+  requestPodboxTicketAction
+} from "@pcd/passport-interface";
+import { expectIsPODTicketPCD } from "@pcd/pod-ticket-pcd";
+import { ONE_DAY_MS, ONE_SECOND_MS } from "@pcd/util";
+import { Identity } from "@semaphore-protocol/identity";
+import { expect } from "chai";
+import "mocha";
+import { step } from "mocha-steps";
+import * as MockDate from "mockdate";
+import { stopApplication } from "../../../../src/application";
+import { PipelineCheckinDB } from "../../../../src/database/queries/pipelineCheckinDB";
+import { PipelineConsumerDB } from "../../../../src/database/queries/pipelineConsumerDB";
+import { PipelineDefinitionDB } from "../../../../src/database/queries/pipelineDefinitionDB";
+import { PipelineUserDB } from "../../../../src/database/queries/pipelineUserDB";
+import { GenericIssuanceService } from "../../../../src/services/generic-issuance/GenericIssuanceService";
+import {
+  PRETIX_CHECKER,
+  PretixPipeline
+} from "../../../../src/services/generic-issuance/pipelines/PretixPipeline";
+import { PipelineUser } from "../../../../src/services/generic-issuance/pipelines/types";
+import { Zupass } from "../../../../src/types";
+import { overrideEnvironment, testingEnv } from "../../../util/env";
+import { startTestingApp } from "../../../util/startTestingApplication";
+import {
+  expectFalse,
+  expectLength,
+  expectToExist,
+  expectTrue
+} from "../../../util/util";
+import {
+  assertUserMatches,
+  checkPipelineInfoEndpoint,
+  proveEmailPCD,
+  requestCheckInPipelineTicket,
+  requestTicketsFromPipeline,
+  signCredentialPayload
+} from "../../util";
+import { setupAutoIssuancePipeline } from "./setupAutoIssuancePipeline";
+
+/**
+ * Tests for {@link GenericIssuanceService}, in particular the {@link PretixPipeline} implementing
+ * auto-issuance features.
+ */
+describe("generic issuance - PretixPipeline", function () {
```

Comment:
Description and the comment above it should say "AutoIssuancePipeline"?

Reply:
done